### ISSUE TITLE
Delete uploaded document

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#10]
## User story
1. As a user
2. I want to be able to delete a document I have already uploaded
3. So that I have control what is stored in app

## Acceptance criteria
* Each individual file should have a delete button to delete that file.
* After deletion, the document no longer shows in the document overview.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
